tests/utils/inputparams/testparams.go

```diff
@@ -43,10 +51,14 @@ func init() {
 func GetVolumeName() string {
 	return volumeName
 }
-
 // GetVolumeNameWithTimeStamp prepares unique volume name by appending current time-stamp value
-func GetVolumeNameWithTimeStamp(volumeName string) string {
-	return volumeName + "_volume_" + strconv.FormatInt(time.Now().Unix(), 10)
+func GetVolumeNameWithTimeStamp(volName string) string {
```

Comment:
@govint `return volName + "_volume_" + strconv.FormatInt(rand.Int63(), 10)` generates volume name uniquely then I am fine to address #1344 as part of this review .. it is just a simple fix and can be accommodated as part of any ongoing review. 

If agree need to remove `GetUniqueVolumeName ` and supply `strconv.FormatInt(rand.Int63(), 10)` to all 3 utils mentioned in #1344.

Please share your idea.